packages/api/src/core/index.ts

```diff
@@ -59,7 +59,7 @@ export default class APICore {
   }
 
   setAuth(...values: string[] | number[]) {
-    this.auth.push(values);
+    this.auth = values;
     return this;
```

Comment:
Not sure why these are all returning `this`. I don't think it needs to, but I kept it to be consistent with the others.

Reply:
I think this was my OOP brain trying to make things chainable and I don't think any of these `set*(` methods need to be.